dbstructure.json

```diff
@@ -94,7 +94,11 @@
 				"admin": {
 					"_uid_": true
 				},
-				"owner": "_uid_"
+				"owner": "_uid_",
+				"codes": {
+					"adminCode": 12345678,
```

Comment:
i think adding `code` after admin and public is redundant because its already in the `codes` object